sdk/mixedreality/Azure.ResourceManager.MixedReality/src/Generated/Models/Serial.cs

```diff
@@ -17,11 +17,11 @@ public enum Serial
         /// The Primary Key
         /// Serialized Name: Serial.1
         /// </summary>
-        Primary,
+        Primary = 1,
```

Comment:
Although this looks like a breaking change, check the [swagger definition](https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/mixedreality/resource-manager/Microsoft.MixedReality/preview/2021-03-01-preview/common.json#L50) and we'll find the previous value is not correct.
Besides, this SDK is not officially released. We can allow the change of enum value.